### ISSUE TITLE
Considerations

### ISSUE THREAD

**Issue by shimanyi** [uonbi/quiz_engine#6]
When a user answers a question, we need to check if he is in probation so that we give him/her a redeem question. Right ? so we have to check more than just the one fact that the user is either registered or not ...

**Comment by janja-janja** [comment]
@shimanyi Not really everytime they submit a question, cuz initially everyone's probation status will be 0 and will only change if they answer 3 incorrect times, so we need to keep count of failures for each user on the submissions table. What's ur thoughts?

**Comment by shimanyi** [comment]
True true, I made some changes on the database,  i should forward it to git. I will do that right away. I have made some changes on the members table then I have added another table for redemptions.

**Comment by shimanyi** [comment]
Do we really need submissions table ?

**Comment by ikosenn** [comment]
@shims it's always good to log stuff and plus I think it can be used to determine the 20 winners

**Comment by ikosenn** [comment]
I don't see the need of a probation table though. It's much easier to add it in the member table since we are not concerned with how many times one is in probation. On member_table  maybe add is_prob and redeem_code.

**Comment by janja-janja** [comment]
@Gray-hat true treu. @shimanyi could you please check this!. ama you're way way deep into probations already ;D

**Comment by shimanyi** [comment]
@Gray-hat  makes sense but I have finished that whole probation table thing :( I was thinking we can add the first 20 guys to the list on the move and just award them. @denisKaranja I had gone too deep with that ...

**Comment by shimanyi** [comment]
It was a job well done guys!
